mountpoint-s3/src/cli.rs

```diff
@@ -1004,4 +1016,21 @@ mod tests {
             parsed.expect_err("invalid bucket name");
         }
     }
+
+    // https://docs.aws.amazon.com/kms/latest/developerguide/concepts.html#key-id
+    #[test_case("arn:aws:kms:eu-west-1:151381207180:key/dabe1478-fe48-47ca-b6f8-ca044b643a82", true; "KMS Key ARN")]
+    #[test_case("arn:aws:kms:us-west-2:111122223333:key/mrk-1234abcd12ab34cd56ef1234567890ab", true; "Multi-region KMS Key ARN")]
```

Comment:
We allow [multi-region key arns](https://docs.aws.amazon.com/kms/latest/developerguide/multi-region-keys-overview.html), but I decided not to add a test for it at this point. 

I've verified locally that a write succeeds for a multi-region key replicated from `us-east-1` to `eu-west-1`.

Reply:
```
$ aws s3api head-object --bucket * --key aaaaaaa2.txt | grep SSEKMSKeyId
    "SSEKMSKeyId": "arn:aws:kms:eu-west-1:*:key/mrk-*",
```